src/main/java/eu/yals/json/ErrorJson.java

```diff
@@ -85,18 +103,10 @@ static Error create(@NotNull ConstraintViolation violation) {
             return error;
         }
 
-        Error message(String errorText) {
+        Error message(final String errorText) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5409395)